draft-ietf-jsonpath-base.md

```diff
@@ -77,16 +77,17 @@ informative:
     target: https://cburgmer.github.io/json-path-comparison/
     title: JSONPath Comparison
     author:
-      name: Christoph Burgmer
+    - name: Christoph Burgmer
```

Comment:
That doesn't look right.

Reply:
Organizations are not shown in references, unless they *are* the author (you now have two authors, one of which is the person, and one is the organization, which is certainly broken).

Reply:
Ok, I've switched to the same format as JSONPath-original (which I've also reverted). I think it's more important that the authors get the credit than their employers.

@cburgmer: please note with my apologies for thinking I could include the organisation in the rendered output.